js/apps/admin-ui/src/context/realm-context/RealmContext.tsx

```diff
@@ -23,7 +23,7 @@ export const RealmContextProvider = ({ children }: PropsWithChildren) => {
 
   const realmParam = routeMatch?.params.realm;
   const realm = useMemo(
-    () => realmParam ?? environment.loginRealm,
+    () => decodeURI(realmParam ?? environment.loginRealm),
```

Comment:
Should this not be using `decodeURIComponent()` instead? Since it's not decoding an entire URL?

Reply:
Good catch, yes, it should